src/Loader.php

```diff
@@ -236,7 +236,7 @@ protected function sanitiseVariableValue($name, $value)
                 %1$s           # match a quote at the start of the value
                 (              # capturing sub-pattern used
                  (?:           # we do not need to capture this
-                  [^%1$s\\\\]* # any character other than a quote or backslash
+                  [^%1$s\\\\]+ # any character other than a quote or backslash
```

Comment:
Why does this need to change please? Why can't we have the empty string?

Reply:
This doesn't need to change, just slightly better regex.  This forces a minimum of 1 characters for tokens not in (\ or (double)quote).  Line 242 ")*" is that actual matching token limit within the quoted value.